src/pages/categories/Categories.jsx

```diff
@@ -16,9 +17,25 @@ const Categories = () => {
     setSearchResults(results)
     setSearchPerformed(true)
   }
+  const isTutor = userRole === 'tutor'
+
   return (
     <PageWrapper>
-      <OfferRequestBlock userRole={userRole} />
+      <HeaderCard
+        buttonText={
+          isTutor
+            ? 'findOffers.offerRequestBlock.button.tutor'
+            : 'findOffers.offerRequestBlock.button.student'
```

Comment:
Since we know that users can be either student or tutor and our translation keys respond to these roles, we can simplify this check:
```javascript
buttonText={`findOffers.offerRequestBlock.button.${userRole}`}
```
or like this, if we don't want to depend on roles:
```javascript
buttonText={`findOffers.offerRequestBlock.button.${isTutor ? 'tutor' : 'student'}`}
```

Same for title.